### PR TITLE
Update for max uncle count and empty steps

### PR DESCRIPTION
**Pull request by Tbaut**
- Fix https://github.com/paritytech/wiki/issues/29

### CODE REVIEW

File: Chain-specification.md [hunk 1]
```diff
@@ -80,6 +80,10 @@ A JSON file which specifies rules of a blockchain, some fields are optional whic
   + `"forkCanonHash"` hash of the canonical block at `forkBlock`
   + `"bombDefuseTransition"` block number at which the difficulty bomb (epsilon in Yellow Paper Eqs. 39, 44) is removed from the difficulty evolution
   + `"wasmActivationTransition"` block number at which bytecode (in storage or transactions) can be run as WASM bytecode and by WebAssembly VM.
+  + `"maximumUncleCountTransition"` Block at which maximum uncle count parameter should be considered. Default `0` for PoA chains, not set for Ethhash chains.
+  + `"maximumUncleCount"` Maximum number of accepted uncles starting from the block specified at `maximumUncleCountTransition`. Default `0` for Ethhash chains, `2` for PoA chains before `maximumUncleCountTransition`.
```
Comment (5chdn) [inline Chain-specification.md]: Ethash with only one H

**Comment by 5chdn** [comment]
can you also add musicoin to the chains in the top section?

File: Chain-specification.md (hunk 1 above)
Comment (Tbaut) [reply Chain-specification.md]: fixed

**Comment by Tbaut** [comment]
oops misclicked

**Pull request merged**